src/main/kotlin/view/TopBar.kt

```diff
@@ -12,26 +12,39 @@ class TopBar : View() {
     private val fileController: FileController by inject()
 
     override val root = menubar {
-        menu("File") {
-            item("Import...") {
+        menu("_File") {
```

Comment:
what does the _ do? will it being displayed in menu?

Reply:
This is used for the `alt` shortcut, e.g. when you press `alt+F` `File` will be clicked.